### DISCUSSION TITLE
简悦 - 讨论组

### DISCUSSION THREAD

**Discussion by Kenshin** [Kenshin/simpread#1506] [讨论组]
## 👋 嗨 ~

欢迎来到 **简悦 · 讨论组**，如果遇到 Bug 的话，请提 [Issues](https://github.com/Kenshin/simpread/issues/new/choose) 这里只是做讨论使用~

**Comment by purequant** [comment]
请问导出为离线markdown能否增加个专门放图片文件夹，然后再从文件夹中引用图片，不喜欢将图片直接存在markdown中，这样我用vscode的mpe插件打开md发现显示不正常，也不利于我对md进行进一步的修改编辑。

**Reply by Kenshin** [reply]
嗨，谢谢建议。未来会考虑导出为 textbundle 结构

**Comment by lzkids** [comment]
我想问一下，怎么下载PC的同步助手？三个链接好像都挂了

**Reply by Kenshin** [reply]
唉，在国内分享个链接太难了。
明天给你个链接吧。另外，你有什么好的推荐？

**Reply by lzkids** [reply]
> 唉，在国内分享个链接太难了。
> 明天给你个链接吧。另外，你有什么好的推荐？

蓝奏云怎么样？虽然我没用过。但我有些软件就是从蓝奏云下载的

**Reply by Kenshin** [reply]
emmm... 可以试试，谢谢提示。

**Reply by Kenshin** [reply]
嗨，我重新分享了百度云的共享 http://ksria.com/simpread/docs/#/Sync?id=下载 你可以试试。
蓝凑云下载的地址，下午再共享下。

**Reply by Kenshin** [reply]
蓝凑云有分享大小的限制，所以我又加了个腾讯微云，还是同样的位置。（估计这会差不多了...

**Reply by strangewalker** [reply]
可以用免费的公用邮箱啊，哈哈! 回到公元前的互联网时代

**Reply by Kenshin** [reply]
@strangewalker

现在应该不会有这个问题了，腾讯微云还挺「靠谱」。

**Comment by lzkids** [comment]
问个问题，电脑重置了。激活插件用的第一次下载的json。但是等激活后想起来有备份的最新的，然后有重新覆盖了一遍。这样算激活了几次？两次还是一次

**Reply by Kenshin** [reply]
同一台设备可以反复激活。
但特殊情况下，重置后会存在设备判断错误的情况，但在「新」设备上仍可以反复激活。

**Comment by jianjiav** [comment]
你好，我想问下如何通过简悦来阅读本地txt？我使用的是最新版的Chrome，但是发现把本地txt拉入Chrome后并不能打开阅读模式。
![image](https://user-images.githubusercontent.com/80157262/110206210-03057d00-7eaf-11eb-8f9d-9b8a5beed87a.png)

**Reply by jianjiav** [reply]
根据官网的图示照做但是发现自己的无法启用阅读模式。

**Reply by Kenshin** [reply]
需要开启个选项，细节看这里

![image](https://user-images.githubusercontent.com/81074/110228316-21b15580-7f3b-11eb-95e2-5edba345d1ab.png)

**Reply by jianjiav** [reply]
已经解决了，非常感谢。

**Reply by Am0731** [reply]
这个打开允许访问文件网址是哪里设置的，发现我的浏览器没有，chrome

**Comment by lzkids** [comment]
无法同步到坚果云，一直显示数据同步中，请稍候... 几个小时还在显示。

**Reply by Kenshin** [reply]
应该不会出现这个问题，看看是否是这个设置造成的 https://github.com/Kenshin/simpread/discussions/1567

**Comment by Am0731** [comment]
大佬，请问有适配小说阅读模式吗?
我阅读小说类网站,进入阅读模式，不能自动翻下一页，查看了插件，插件不能自由添加适配的网站，所以我的网站不能适配。

**Reply by Kenshin** [reply]
嗨，小说的话，有个油猴脚本，更适合使用，你去翻翻小众那边，我记得有个介绍。

PS：简悦也能做到这些，但因为已经有了不错的方案了，我就没打算「重复造轮子」。

更正下：简悦可以自由添加适配站点，这点你应该是理解错了（或找错地方了。

**Comment by MonicaIII** [comment]
![photo_2021-04-09_16-08-55](https://user-images.githubusercontent.com/36149366/114149944-fd1a1600-990a-11eb-8cd6-a74046d26237.jpg)
保存后在onenote找不到

**Reply by Kenshin** [reply]
Onenote API 暂时有些问题（ 但我没有保存失败过...  😅  确实有用户提过这个问题。

不过，下个版本已解决了这个问题，暂时没发布，发布后请留意下。

**Comment by i123456139** [comment]
我想问下，用简悦 保存到notion，提示我保存成功，可是我去notion查看，根本就没有生成新的page。是需要设置什么吗。非常无语，在帮助文件，也没有讲是不是需要设置，总之感觉让我觉得很难理解，为什么没有这个介绍说明。

**Reply by ghost** [reply]
您好，这个问题你解决了吗？我现在也是这个情况，导出到notion后的位置是随机的，在某一个page尾部，不知道怎么固定导出后的位置，不然太难找了。

**Reply by Kenshin** [reply]
@hongjunyang 

导入到 `database  page` 根据时间排序，如下图所示

![image](https://user-images.githubusercontent.com/81074/122746401-8dd28200-d2bc-11eb-8d22-2b9b64161320.png)

**Reply by ghost** [reply]
谢谢

**Comment by strangewalker** [comment]
单一电脑，在Win64下的edge(dev版）已经成功注册了高级帐户，请问同一电脑帐号下，其它浏览器，如edge,brave...能否使用高级帐户权呢？

**Reply by Kenshin** [reply]
可以的。

同一设备在 2.1.0 下是不限浏览器的（ 2.0.0 有个 Bug 导致会存在问题）

但如果是多显示器的话，偶尔会发生误判。

**Reply by strangewalker** [reply]
TKS，已经用dropbox同步过来了。Firefox版本的V2版扩展啥时候能用上啊？

**Reply by Kenshin** [reply]
@strangewalker

搞定就好，FF 版短期内不会考虑了，但肯定会上架的。（至于原因的话，好奇就去 TG 群里翻翻。  😂

**Comment by hbfrank** [comment]
已购买并激活高级用户，但是在阅读模式导出PDF时提示“当前用户不是高级账户”，如下图所示：
<img width="366" alt="Snipaste_2021-04-17_01-27-19" src="https://user-images.githubusercontent.com/17422123/115061860-42969e80-9f1c-11eb-8d9b-1d026c6ba163.png">
<img width="452" alt="Snipaste_2021-04-17_01-27-03" src="https://user-images.githubusercontent.com/17422123/115061885-49bdac80-9f1c-11eb-93c1-360fece6fa76.png">

**Reply by Kenshin** [reply]
嗨，这是个 Bug 解决方案请看 https://github.com/Kenshin/simpread/discussions/1562 https://github.com/Kenshin/simpread/discussions/1566

**Comment by jxyddtc** [comment]
可以设置不弹出【请确保已运行简悦·同步助手】吗？我只有要用的时候才会打开它，并不想时时刻刻被提醒......

**Reply by Kenshin** [reply]
> 可以设置不弹出【请确保已运行简悦·同步助手】吗？我只有要用的时候才会打开它，并不想时时刻刻被提醒......

程序上是没问题的，但这个提示是为了 **当开启某些功能后却不运行它，导致无法使用某些功能** 而设置的。

我考虑下，谢谢建议。

**Comment by minclearlove** [comment]
为什么支付宝付了款，但没有收到密钥，也找不到选项页

**Reply by Kenshin** [reply]
嗨，支付直接付款时，已经说明了 **不是自动兑换，需要 24 ~ 48 小时**

![image](https://user-images.githubusercontent.com/81074/116516446-35a48280-a900-11eb-942c-0b5756faf301.png)


***

> 找不到选项页

简悦 icon 鼠标右键 → 选项

**Comment by minclearlove** [comment]
订单号填错了咋办

**Reply by Kenshin** [reply]
重新填写

**Comment by lfk2458** [comment]
请问，以后会支持查看别人分享的主题吗？

**Reply by Kenshin** [reply]
嗨，在 https://github.com/Kenshin/simpread/issues/71 回复你了。

**Comment by zllebron3wade** [comment]
添加 @simpread_bot
使用 /token xxxx 来绑定你的 Token
发送包含 URLs 的内容给它，它就会自动帮你整理到网盘             请问网盘在哪里？没懂啊，求详细告知

**Reply by Kenshin** [reply]
看看 https://simpread.pro/api/#/如何开通 （在 [开放平台](https://simpread.pro/developer) 就有这个链接）

**Reply by zllebron3wade** [reply]
我根据这个步骤：简悦 → 选项页 → 共通 → 简悦 · 同步助手 → 授权验证并开启简悦 · 同步助手       点了授权验证发现没反应，多点几次以后，右上角弹出红色提示，请确保已运行同步助手。

**Reply by zllebron3wade** [reply]
原来是要安装同步助手的pc端软件，我个人觉得你们的教程写的不清晰= =

**Reply by zllebron3wade** [reply]
我安装好了，然后同步json的路径和output文件路径也都建立了，我通过发网址给telegram bot，最后生成到哪个目录？为什么我测试了很多条，一个都没有

**Reply by Kenshin** [reply]
![image](https://user-images.githubusercontent.com/81074/117536532-17d8cb00-b02e-11eb-92b0-3f2d856918ee.png)

**Comment by lzkids** [comment]
如何合并标注？

**Comment by zach14c** [comment]
更新到2.2後,相當困擾,每一個網頁都會進入閱讀模式,不管是有沒有加黑名單,白名單,完全沒有作用,相當困擾,只能先disable它.

**Reply by Kenshin** [reply]
嗨，这是解决方案 https://github.com/Kenshin/simpread/issues/2159

**Reply by zach14c** [reply]
可以了,謝謝.

**Comment by clfvz** [comment]
請問為什麼不支援 Facebook 閱讀?

**Reply by Kenshin** [reply]
给个 URL 看看

**Reply by Kenshin** [reply]
嗨，抱歉，今天一直在处理文档的事情，耽搁了。

- 简悦 2.2.0 文档已全部更新，细节请看 https://github.com/Kenshin/simpread/issues/2158#issuecomment-845868642

- 简悦 2.2.0 发布后产生了一些通用性问题，细节请看 https://github.com/Kenshin/simpread/issues/2158

**Comment by LoneFireBlossom** [comment]
问一下，我提交了「只要你的捐助价格超过（含） 8 元，即可通过此 联系方式 获得简悦的终身高级账户。」这个的凭证，等了可能一个多月了吧，咋还没回复呢emmm。
我的邮箱是llonefireblossom@gmail.com，就是用这个邮箱地址提交的。

**Reply by Kenshin** [reply]
请问你是什么时候提交的？另外，如果是 2.0 之后提交的将不会享受此方式

![image](https://user-images.githubusercontent.com/81074/128006663-5f8361c2-764a-4a41-bd3f-5fc0ef9c49cd.png)

**Reply by LoneFireBlossom** [reply]
2020.3.16的大概。
讨论链接：
https://github.com/Kenshin/simpread/issues/677#issuecomment-599993542

**Reply by Kenshin** [reply]
嗨，时间上是没问题的，近期在休假，明天我确认下。

如果你方便的话，把购买凭证发到 Github 我的邮箱里面也行。🙏

**Reply by LoneFireBlossom** [reply]
已发👌

**Reply by Kenshin** [reply]
嗨，谢谢。

我看到了，不过能否有之前请咖啡时的订单截图？

**Reply by LoneFireBlossom** [reply]
已发👌

**Reply by Kenshin** [reply]
嗨，看到截图了。

明天有时间给你 License 🙏 （ 因近期在休假，无法 100% 确定时间）

**Reply by Kenshin** [reply]
嗨，你的 License 已通过邮箱发过去了，谢谢之前的请杯咖啡。

**Reply by LoneFireBlossom** [reply]
收到，感谢🙏

**Comment by BasaNite16** [comment]
不知道能否分别对中英文文章设置不同字体。中文用的是思源屏显宋,但又常常阅读英文文章,在简悦上显示效果不理想,所以一直以来只能用edge自带的阅读器。

**Reply by Kenshin** [reply]
应该是可以的，你可以提供一些英文字体比较好看的页面供我参考。

**Reply by BasaNite16** [reply]
[TIME](https://time.com/6088769/covid-19-vaccinations-rising-fourth-wave/)
[Thrive](https://thrive.design/5-website-maintenance-tips-you-need-to-know/)
[Reuters](https://www.reuters.com/world/us/mississippi-hospital-puts-beds-parking-garage-cope-with-covid-19-surge-2021-08-13/)
[AP News](https://apnews.com/article/joe-biden-business-taliban-f600d6faf28e9c2ccb454ad176987b19)
[NEJM](https://www.nejm.org/doi/full/10.1056/NEJMoa2109522?query=featured_home)
[The Atlantic](https://www.theatlantic.com/ideas/archive/2021/08/schwarzenegger-schmuck-mask-vaccines/619746/)
纯粹就个人喜好（毕竟字体太多了），比较倾向衬线中 Georgia，PT Serif，易读性挺好; 非衬线字体像经典的Arial，Lato，Verdana 也不错。

**Reply by Kenshin** [reply]
收到，回头研究下~

**Reply by Kenshin** [reply]
嗨，这个问题解决了，你可以安装 [英文阅读版面辅助增强](https://github.com/Kenshin/simpread/issues/2840)。

![image](https://user-images.githubusercontent.com/81074/133365550-ba9d9bd6-3d27-4524-8af4-984e48e7387a.png)

> 同时也可以使用下面两个跟英文阅读有关系的内容：

- [全文翻译](https://simpread.ksria.cn/plugins/details/Y7JxbP7B4H)

- [适配扇贝阅读的适配站点](https://github.com/Kenshin/simpread/discussions/2705)

**Reply by BasaNite16** [reply]
感谢 Kenshin，插件效果很不错。

**Reply by Kenshin** [reply]
> 感谢 Kenshin，插件效果很不错。

哈哈，喜欢就好。

因为一直没想好用什么方式实现，所以拖到了现在...  😂

**Reply by Kenshin** [reply]
@BasaNite16

嗨  https://simpread.ksria.cn/plugins/details/3PHAZerSkb 也已经更新完毕，另外我整理了一份关于英文阅读的合辑，你可以看下这里 https://zhuanlan.zhihu.com/p/410684466

**Reply by BasaNite16** [reply]
哈哈，已经好评。现在阅读英文内容顺畅多了。

**Reply by Kenshin** [reply]
> 哈哈，已经好评。现在阅读英文内容顺畅多了。

嘿嘿，必须地~

**Comment by cescbamboo** [comment]
想问一下阅读模式下，图片的显示尺寸可以调整吗？因为当前图片显示的尺寸较大，对于那种图片比较多的图文并茂的文章，会导致图片之间的文字变得不明显。或者说鼠标拖过去的时候图片占据画面比较大导致寻找文字相对不那么容易了一些。

**Reply by Kenshin** [reply]
嗨，目前没有相关功能。不过，可以考虑你的建议，但最好给我个 URL 看看。

另外，建议发个 Issues https://github.com/kenshin/simpread/issues/new 并详细说明你的诉求。😄

**Reply by cescbamboo** [reply]
> 嗨，目前没有相关功能。不过，可以考虑你的建议，但最好给我个 URL 看看。
> 
> 另外，建议发个 Issues https://github.com/kenshin/simpread/issues/new 并详细说明你的诉求。😄

感谢哈，没直接发 Issues 是怕可以设置但是我没找到:laughing:​ 现在发好啦 https://github.com/Kenshin/simpread/issues/3982

**Reply by Kenshin** [reply]
okay 我已经记录你的功能建议~

**Comment by 082333** [comment]
你好，我想问下简悦的站点编辑器要怎么改呢？我把某个网站的文章收集保存为稍后阅读后，那个标题全部都是统一 一样的，而不是每一篇文章的标题，这样看文章都不知道哪一篇了

**Comment by 0ZDS** [comment]
有什么操作可以在打开网站后，直接看到批注吗。
目前我说看到的是需要打开阅读模式，并且开启批注显示才可以看到。

**Reply by Kenshin** [reply]
有深度阅读必要的时候才会使用简悦的标注，简悦是专业系统的信息管理工具，在原网页仅标注一段内容的话，有很多其他工具。

**Comment by 0ZDS** [comment]
好的，了解，感谢回复



在 2022-08-12 10:30:01，"Kenshin Wang" ***@***.***> 写道：

有深度阅读必要的时候才会使用简悦的标注，简悦是专业系统的信息管理工具，在原网页仅标注一段内容的话，有很多其他工具。

**Comment by 8kita** [comment]
在微信公众号使用简悦保存到notion，可以加上文章发布的时间吗><

**Reply by Kenshin** [reply]
哈，你是说发布到 Notion 的时间？还是指什么？如果是前者的话，导入后就有时间了（是指导入的时间），细节可以看 http://ksria.com/simpread/docs/#/Notion

**Reply by 8kita** [reply]
是所导入文章在微信公众号发布的时间

**Reply by Kenshin** [reply]
> 是所导入文章在微信公众号发布的时间

暂时没有这种功能。

不过，会考虑下的。

**Reply by 8kita** [reply]
好的谢谢您

**Reply by Kenshin** [reply]
小事情，十一长假多休息~

**Comment by Ceritor-Hanio** [comment]
请问我能不能在将一篇文章加入稍后读的同时顺手给他加标签呢？因为我经常遇到突然看到好文章没时间读，而后面想要一口气读同类文章的情况。
要是能在加入的时候顺手分类，就会方便很多。

**Reply by Kenshin** [reply]
哈，已经在 todo 里面了。

**Reply by Ceritor-Hanio** [reply]
加油！期待简阅变得更好！

**Comment by FCJ1987** [comment]
请问能否使用多个简悦账号在同一个坚果云里同步？
我和朋友共用了同一个坚果云账号，以便在一些需要协作的工作中共同分享和编辑。所以我们在坚果云里将文件分成了A（我）/B（对方）/公用等不同的文件目录，按需同步。其中A/B文件夹为各自编辑，可以共同查看但是不会修改对方的内容，而公用文件则是既会共同查看也会共同编辑。
按简悦同步助手的说明，是在坚果云的根目录内建立SimpRead/output文件夹的话，如果我和对方各有各的简悦账号，同时还有一个公用账号，这种情况下能否使用多个简悦账号在同一个坚果云上进行同步？同步文件夹是必须在根目录下的唯一路径吗，能否通过在坚果云里建不同的文件夹目录来区分不同的简悦账号同步内容？

**Reply by Kenshin** [reply]
只要不使用 API 的话，将 SimpRead 放到坚果云哪个位置都行，比如：

1.  `<坚果云根目录>/A/SimpRaed/`

2. `<坚果云根目录>/B/SimpRaed/`

但这样一来，无法使用 API 相关功能，因为API 相关功能只能在根目录下，且也跟坚果云账户有关系。

PS：不使用 API 的话，用什么网盘也都无所谓。

**Comment by tyskink** [comment]
刚刚订阅高级账户
1. 同一台设备的两个浏览器是使用同一个配置文件比较好，还是再买一个高级账户比较好？
2. 两台设备都要使用简悦，购买两个高级账户，如何保持同步能力？
3. 使用简悦打开小说网站，但是点一下下一页/下一章就退出阅读模式了，不够连贯，这个能够解决吗？

**Reply by Kenshin** [reply]
> 问题1和2

它们是一个问题，你在同一个设备使用不同的浏览器算是一台设备。（极个别情况下会被误判，但目前发生此问题的几乎忽略不记）

让同一个设备的不同浏览器保持同步的话，需要安装同步助手，教程可以 [看这里](https://www.yuque.com/kenshin/simpread/pwpnsx)。

> 问题3

简悦不是专门用来看小说的，看小说的有很多不错的油猴脚本（专门针对小说类的网站做了优化），但你非要使用简悦的话，请安装 [自动翻译插件](https://simpread.ksria.cn/plugins/details/QY0PepohmP)

PS：再有任何问题的话，可以通过 https://github.com/kenshin/simpread/issues/new 讨论。（不要在这里讨论）

**Comment by XMB-7** [comment]
话说不考虑给简悦同步助手改个名么, 在功能上属于以偏概全, 还是比较有误导性的. 
改成简悦桌面助手或者干脆就叫简悦助手, 都会更加合理一些. 
当然这也不是什么高优先级的事情, 只不过修改文档之类会比较繁琐.

**Reply by Kenshin** [reply]
哈，同步助手这个名字，你可以理解为：负责「同步」的助手。

同步的范畴包括：配置文件（含稍后读等）、导出到 xxx 服务，都可以算做同步的范畴。 🌚 但，目前加入的新功能方面确实超出了这个范畴。

会考虑改名的  😂

**Reply by XMB-7** [reply]
主要是随着功能的增加, 即便是再外扩同步的含义, 也越来越难以覆盖住全部的功能, 而新用户看到同步助手这一个名字, 也往往压根想不到其背后所蕴含的丰富功能. 
简悦助手或者简悦伴侣这种(simpread helper/assistant/companion/enhancer...)更广泛的名字会越来越合适.

**Comment by XMB-7** [comment]
请问阅读模式页尾信息能自定义么, 比如隐藏或者修改. 
能否在扩展的配置页面里增添相应的文本模板, 或提供相应的插件.
![image](https://github.com/user-attachments/assets/a5d911af-3a40-40d7-96c8-53dfdd3c70c9)

**Reply by Kenshin** [reply]
哈，很久没看到你提 Issues 了~

小长假快乐 😄

可以通过定制 CSS 来解决（也就是把它隐藏掉），但不支持其它形式的定制。

```
sr-rd-footer {display:none}
```

加入到全局自定义样式即可。

**Reply by XMB-7** [reply]
节假快乐. 最近又有时间和闲心了, 就想着往回捡一捡知识管理相关的软件使用.
用css的问题是就只能在阅读时隐藏页脚, 但似乎并不能在导出的本地快照里也将之隐藏
因此最好是可以在同步助手的导出功能增强部分, 再新增一个不导出页脚的选项.

**Reply by Kenshin** [reply]
这个暂时不能，我会考虑下的~

**Comment by Lokkue** [comment]
请问简悦同步助手支持同步稍后读嘛？😕
我在局域网下有两个电脑可以用Syncthing同步文件夹，如果能将稍后读的保存路径通过同步助手自定义的话应该就能实现同步。
（或者其实有类似功能，我没找到？💦）

**Reply by Kenshin** [reply]
当前可以，建议看下https://www.yuque.com/kenshin/simpread/wkswh7 教程随时是坚果云，但任意网盘都可以，将坚果云的替换为你自己的 Syncthing 即可。

PS：猜测你是喜欢折腾的用户，感兴趣的话可以看下 https://www.yuque.com/kenshin/simpread/dgmm5q 里面有多设备同步的一些实现逻辑，会对你折腾更有帮助 😄

PPS：简悦相比其它其它产品最好的优势在于：数据本地化（类似 Obsidian ），所以多设备同步是它最基本的功能。（在同步助手的加持下）

**Comment by Alexstyhuang** [comment]
请问TG讨论群之前没有入群验证，被封了，怎么解封啊

**Reply by Kenshin** [reply]
提供 id 给我。

另外，当被封时会自动告诉你管理员，你可以直接联系管理员，而不是在这里 🤣
